### PR TITLE
layer for navigation [op-54]

### PR DESCRIPTION
**Pull request by Rytiggy**

### CODE REVIEW

File: app/components/layer-settings/style.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
     color: #fff;
     position: relative;
     height: 42px;
+    z-index: 10;
```
Comment (caneruguz) [inline app/components/layer-settings/style.scss]: Why was this needed?

File: app/components/layer-wrapper/component.js [hunk 2]
```diff
@@ -16,6 +16,18 @@ export default Ember.Component.extend({
         let height = this.get('layer.settings.values.height') ? 'height: ' + this.get('layer.settings.values.height') + 'px;' : '';
         return Ember.String.htmlSafe(bgImage + bgCover + height + bgColor + color);
     }),
+    didRender(){
+        let self = this;
+        $.each(this.get('layers'), function( index, layer ) {
+            if(layer.component === 'pages-menu'){
+                $('.pages-menu' ).find('.layer-settings').css({
```
Comment (caneruguz) [inline app/components/layer-wrapper/component.js]: Why is this adjustment necessary?

File: app/components/pages-menu/template.hbs [hunk 3]
```diff
@@ -10,4 +10,7 @@
     </ul>
 </div>
 
+<div>
+</div>
```
Comment (caneruguz) [inline app/components/pages-menu/template.hbs]: Should this be removed?

File: app/components/pages-menu/style.scss [hunk 4]
```diff
@@ -6,3 +6,23 @@
     border-radius: 2px;
     text-decoration: none;
 }
+
+.pages-nav {
+    &.left {
+       text-align: left;
+       float:left
```
Comment (caneruguz) [inline app/components/pages-menu/style.scss]: Are the floats necessary?

File: app/components/add-layer/component.js [hunk 5]
```diff
@@ -8,173 +8,209 @@ export default Ember.Component.extend({
...
+                            "right"
+                            ],
+                            "validation": null
                         }
-                    };
-                    break;
+                        ]
+                    }
+                };
+                break;
```
Comment (caneruguz) [inline app/components/add-layer/component.js]: This file is mostly spacing changes right?

File: app/components/layer-settings/style.scss (hunk 1 above)
Comment (Rytiggy) [inline app/components/layer-settings/style.scss]: it is needed because if the user makes the layer to small the settings would be hidden behind the layer below making that layer unusable

File: app/components/layer-wrapper/component.js (hunk 2 above)
Comment (Rytiggy) [inline app/components/layer-wrapper/component.js]: when you go in to edit mode the settings appears above the navigation, pushing it below the layer view, this is here so you can see the changes that you are making on the nav

File: app/components/pages-menu/style.scss (hunk 4 above)
Comment (Rytiggy) [inline app/components/pages-menu/style.scss]: In testing, text-align did not move the navigation, floating was the only way I could get it to move

File: app/components/add-layer/component.js (hunk 5 above)
Comment (Rytiggy) [inline app/components/add-layer/component.js]: Yeah I added the ability to add navigation in add-layer

File: app/components/pages-menu/style.scss (hunk 4 above)
Comment (caneruguz) [inline app/components/pages-menu/style.scss]: It's probably because the size of the navigation element did not extend to full width. Can you try to find out why align is not working, because that should be the right way to do it.
